dashboard/src/lib/component/base/Duration/Duration.js

```diff
@@ -18,14 +18,16 @@ class Duration extends React.PureComponent {
       // ex. 892ms
       const formatter = this.numberFormatter({ maximumFractionDigits: 0 });
       return `${formatter.format(duration)}ms`;
-    } else if (duration < 10000) {
+    }
+    if (duration < 10000) {
       // ex. 8.02s
       const formatter = this.numberFormatter({
         minimumFractionDigits: 2,
         maximumFractionDigits: 2,
       });
       return `${formatter.format(duration / 1000)}s`;
-    } else if (duration < 60000) {
+    }
+    if (duration < 60000) {
```

Comment:
Should this remain an if/else if? Otherwise we will run both of these blocks if `duration is < 6000` - and it looks to me as though it overwrites what occurs in the first block. It still gives the correct results, but it's just extra cycles.

Reply:
Oh wait, sorry I had my signs backwards

Reply:
Wait, it still stands. Sorry, still working through my tea this morning!

Reply:
I believe it's the [`no-else-return`](https://eslint.org/docs/rules/no-else-return#allowelseif-false) rule that made this change. It's a bit odd at first glance but in the interest of consistency I kept it.

Reply:
This feels wrong to me, but if consistency is that important...

Reply:
I didn't see there was a return in the statement, retract my comment.